### PR TITLE
Fix wrongly overrided toString function from Function constructor

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

File: src/client/sandbox/node/window.js [hunk 1]
```diff
@@ -237,9 +237,10 @@ export default class WindowSandbox extends SandboxBase {
 
         window.Function.prototype             = nativeMethods.Function.prototype;
         window.Function.prototype.constructor = window.Function;
-        window.Function.toString              = function () {
-            return nativeMethods.Function.toString();
-        };
+
+        // NOTE: We need to create function which returns string without calling toString every time
+        // because if the Function.prototype.toString is overridden it can be the cause of recursion
+        window.Function.toString = (nativeStr => () => nativeStr)(nativeMethods.Function.toString());
```
Comment (miherlosev) [inline src/client/sandbox/node/window.js]: It is difficult to read.
Something like
```
const nativeFunctionToString = nativeMethods.Function.toString();

window.Function.toString = () => nativeFunctionToString;

```

File: test/client/fixtures/sandbox/node/window-test.js [hunk 2]
```diff
@@ -137,4 +137,19 @@ test('the constructor field of a function should return a wrapped Function objec
 
     strictEqual(f.constructor, Function);
     strictEqual(f.constructor.toString(), nativeMethods.Function.toString());
+
+    var toString       = Function.toString;
+    var nativeToString = Function.prototype.toString;
+
+    /*eslint-disable no-extend-native*/
+    Function.prototype.toString = function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/window-test.js]: If I comment your fix in code then this test will be green.

Comment (LavrovArtem) [inline test/client/fixtures/sandbox/node/window-test.js]: image

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping @churkin

**Pull request merged**